examples/getting_started/src/lib.rs

```diff
@@ -5,13 +5,15 @@
 pub mod butane_migrations;
 pub mod models;
 
+use butane::_filenames::{BUTANE_DIRNAME, CONNECTION_JSON_FILENAME};
 use butane::db::{Connection, ConnectionSpec};
 use butane::prelude::*;
 use models::{Blog, Post};
 
 /// Load a [Connection].
 pub fn establish_connection() -> Connection {
-    butane::db::connect(&ConnectionSpec::load(".butane/connection.json").unwrap()).unwrap()
+    let connspec = format!("{BUTANE_DIRNAME}/{CONNECTION_JSON_FILENAME}");
```

Comment:
IMO the fact that the examples need to include these strings feels like a problem.
These shouldnt need to be so visible - they should be a hidden detail that are only important for unusual usage.

Reply:
I think `ConnectionSpec::load` should auto detect if this filename exists